### ISSUE TITLE
MoistAir: Temperature validity range seems to be wrong

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1112]
**Reported by otter on 20 May 2013 15:15 UTC**
The MoistAir Medium model has the following definition 

```
package MoistAir "Air: Moist air model (190 ... 647 K)"
  extends Interfaces.PartialCondensingGases(
    mediumName="Moist air",
    substanceNames={"water","air"},
    final reducedX=true,
    final singleState=false,
    reference_X={0.01,0.99},
    fluidConstants={IdealGases.Common.FluidData.H2O,IdealGases.Common.FluidData.N2},
    Temperature(min=200, max=423.15));

```
The description text states a validity range of 190 - 647 K, but the min/max values in Temperature state a validity range of 200 - 423.15. This seems to be wrong

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1112

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 20 May 2013 15:17 UTC**
Fixed in 78ecfd35e93aebeb312f120cab26f4d156e3be55 by changing the modifier of Temperature to:

```
Temperature(min=190, max=647)
```

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 20 Dec 2016 15:11 UTC**
